src/gps/GPS.cpp

```diff
@@ -999,7 +981,7 @@ int32_t GPS::runOnce()
                       GNSS_MODEL_UBLOX10)) {
             // reset the GPS on next bootup
             if (devicestate.did_gps_reset && scheduling.elapsedSearchMs() > 60 * 1000UL && !hasFlow()) {
-                LOG_DEBUG("GPS is not communicating, trying factory reset on next bootup.");
+                LOG_DEBUG("GPS is not communicating, trying factory reset on next boot.");
```

Comment:
Should we get rid of periods at the end of sentences as well? :-)

Reply:
You joke, but that probably represents nearly as much flash as the \n purge.

Reply:
I'm actually not joking. I don't need proper punctuation in my logs. Haha